test/resources.test.js

```diff
@@ -573,27 +477,27 @@ describe('Resources query', function () {
   describe('itemsQueryContext', () => {
     it('should ignore check in card items when merge_checkin_card_items is not set', () => {
       expect(resourcesPrivMethods.itemsQueryContext({}))
-        .to.deep.equal({ must_not: [{ exists: { field: 'items.electronicLocator' } }, { term: { 'items.type': 'nypl:CheckinCardItem' } }] })
+        .to.deep.equal({ must_not: [{ term: { 'items.type': 'nypl:CheckinCardItem' } }] })
     })
 
     it('should ignore check in card items when merge_checkin_card_items is falsey', () => {
       expect(resourcesPrivMethods.itemsQueryContext({ merge_checkin_card_items: false }))
-        .to.deep.equal({ must_not: [{ exists: { field: 'items.electronicLocator' } }, { term: { 'items.type': 'nypl:CheckinCardItem' } }] })
+        .to.deep.equal({ must_not: [{ term: { 'items.type': 'nypl:CheckinCardItem' } }] })
     })
 
     it('should include check in card items when merge_checkin_card_items is truthy', () => {
       expect(resourcesPrivMethods.itemsQueryContext({ merge_checkin_card_items: true }))
-        .to.deep.equal({ must_not: [{ exists: { field: 'items.electronicLocator' } }] })
+        .to.deep.equal({ must: { match_all: {} } })
```

Comment:
This and the next two tests don't obviously assert what the `it` string says. To be fair, it doesn't look like they did before either. Maybe some comments would help explain? In particular the test on line 498 calls out electronic items, but doesn't appear to assert their absence in anyway.

Reply:
Good catch. Thanks for reading closely. I looked more closely at these tests. A couple were using config we no longer support. I updated the `it` statement to better reflect what we're testing.